docs/product/ai-monitoring/getting-started/the-dashboard.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: The Dashboard
+sidebar_order: 100
+---
+
+
+Once you've [configured the Sentry SDK](/product/ai-monitoring/getting-started/) for your AI project, you'll start
+receiving data in the Sentry AI Monitoring dashboard.
```

Comment:
```suggestion

Once you've configured the [Sentry SDK](/product/ai-monitoring/getting-started/) for your AI project, you'll start
receiving data in Sentry's AI Monitoring dashboard.
```

Reply:
Changing the link like this makes it less clear that the link is to the configuration page - I would expect [sentry sdk] to just be the homepage of the docs or something like that.